matRadGUI.m

```diff
@@ -26,13 +26,13 @@
 
```

Comment:
porbably "best" practice of my editor. However maybe we should clean up the whole project and remove trailing whitespace (e.g. using https://github.com/zhengjia/rstrip) and have one "whitespace" commit?